archive/j/javascript/README.md

```diff
@@ -9,6 +9,7 @@ Welcome to Sample Programs in JavaScript!
 - [Fizz Buzz in JavaScript][4]
 - [Baklava in JavaScript][8]
   - Solution borrowed from @toturkmen via the [baklava repo][1]
+- [Fibonacci in JavaScript]
```

Comment:
Looks good! Can you make the article issue and link it in the README? There are examples in the README.